phys2bids/utils.py

```diff
@@ -27,12 +61,23 @@ def path_exists_or_make_it(fldr):
         os.makedirs(fldr)
 
 
-def check_file_exists(file, hardexit=True):
+def check_file_exists(filename):
     """
     Check if file exists.
     """
-    if not os.path.isfile(file) and file is not None:
-        raise FileNotFoundError('The file ' + file + ' does not exist!')
+    if not os.path.isfile(filename) and filename is not None:
+        raise FileNotFoundError(f'The file {filename} does not exist!')
+
+
+def print_info(filename, phys_object):
+    """
+    Print the info of the input files, using BlueprintInput object
+    """
+    print(f'File {filename} contains:\n')
+
+    for ch in range(2, phys_object.ch_amount):
+        print(f'{(ch-2):02d}. {phys_object.ch_name[ch]};'
+              f' sampled at {phys_object.freq[ch]} Hz')
```

Comment:
Shouldn't this be a method inside ``BlueprintInput``?

Reply:
We can move it there, yes.